tesseract/tesseract_planning/tesseract_command_language/include/tesseract_command_language/command_language_utils.h

```diff
@@ -196,9 +200,29 @@ std::vector<std::reference_wrapper<Instruction>> flattenToPattern(CompositeInstr
  * vector
  * @return A new flattened vector referencing the original instruction elements
  */
-std::vector<std::reference_wrapper<const Instruction>> flattenToPattern(const CompositeInstruction& instruction,
-                                                                        const CompositeInstruction& pattern,
-                                                                        bool process_child_composites = false);
+std::vector<std::reference_wrapper<const Instruction>>
+flattenToPattern(const CompositeInstruction& composite_instruction,
+                 const CompositeInstruction& pattern,
+                 bool process_child_composites = false);
```

Comment:
parameter `process_child_composites` doesn't match param in description, `include_composite`

Reply:
There are a few instances of this, not sure if they're effectively supposed to mean the same thing or if the parameter is misidentified

Reply:
Unless it has changed, process_child_compisites is a misnomer. It used to specify where or not the raw composite was stored in the result (so that  you could access things like start_instructions). Either way the children get processed (since that is what flatten means).

Reply:
I am restructuring these utility function to make them generic where you can provide a function callback to indicate if something should be kept or not. Then I will create one that are specific to motion planning in the planning utilities.